templates/admin/user/view_details.tmpl

```diff
@@ -0,0 +1,71 @@
+<div class="flex-list">
+	<div class="flex-item">
+		<div class="flex-item-leading">
+			{{ctx.AvatarUtils.Avatar .User 48}}
+		</div>
+		<div class="flex-item-main">
+			<div class="flex-item-title">
+				{{template "shared/user/name" .User}}
+				{{if .User.IsAdmin}}
+					<span class="ui basic label">{{ctx.Locale.Tr "admin.users.admin"}}</span>
+				{{end}}
+			</div>
+			<div class="flex-item-body">
+				<b>{{ctx.Locale.Tr "admin.users.auth_source"}}:</b>
+				{{if eq .LoginSource.ID 0}}
+					{{ctx.Locale.Tr "admin.users.local"}}
+				{{else}}
+					{{.LoginSource.Name}}
+				{{end}}
+			</div>
+			<div class="flex-item-body">
+				<b>{{ctx.Locale.Tr "admin.users.activated"}}:</b>
+				<strong>
+					{{if .User.IsActive}}
+						{{svg "octicon-check"}}
+					{{else}}
+						{{svg "octicon-x"}}
+					{{end}}
+				</strong>
+			</div>
+			<div class="flex-item-body">
+				<b>{{ctx.Locale.Tr "admin.users.restricted"}}:</b>
+				<strong>
+					{{if .User.IsRestricted}}
+						{{svg "octicon-check"}}
+					{{else}}
+						{{svg "octicon-x"}}
+					{{end}}
+				</strong>
+			</div>
+			<div class="flex-item-body">
+				<b>{{ctx.Locale.Tr "settings.visibility"}}:</b>
+				{{if .User.Visibility.IsLimited}}{{ctx.Locale.Tr "settings.visibility.limited"}}{{end}}
+				{{if .User.Visibility.IsPrivate}}{{ctx.Locale.Tr "settings.visibility.private"}}{{end}}
+			</div>
+			<div class="flex-item-body">
+				<b>{{ctx.Locale.Tr "admin.users.2fa"}}:</b>
+				<strong>
+					{{if .TwoFactorEnabled}}
+						<span class="text green">{{svg "octicon-check"}}</span>
+					{{else}}
+						{{svg "octicon-x"}}
+					{{end}}
+				</strong>
+			</div>
+			{{if .User.Location}}
+				<div class="flex-item-body">
+					<span class="flex-text-inline">{{svg "octicon-location"}}{{.User.Location}}</span>
+				</div>
+			{{end}}
+			{{if .User.Website}}
+				<div class="flex-item-body">
+					<span class="flex-text-inline">
+						{{svg "octicon-link"}}
+						<a target="_blank" rel="noopener noreferrer me" href="{{.User.Website}}">{{.User.Website}}</a>
```

Comment:
> The rel="me" attribute is used in RelMeAuth and Web sign in specifications as a way to enable a person to identify themselves to a web service using their domain name or a particular URL.

It doesn't seem right to use `me`

And, `noopener noreferrer` seems both no-op. 

* noopener: it's the default value for `target=_blank`
* noreferrer: the page head alread declares it:  `<meta name="referrer" content="no-referrer">`